lib/ansible/modules/windows/win_eventlog_entry.ps1

```diff
@@ -0,0 +1,121 @@
+#!powershell
+
+# (c) 2017, Andrew Saraceni <andrew.saraceni@gmail.com>
```

Comment:
The copyright is a lot simpler now, you only need a few lines. See http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#copyright for what has changed.